lib/perl/Genome/Model/SomaticValidation/Command/AlignmentStatsSummary.pm

```diff
@@ -143,7 +143,13 @@ sub _alignment_metrics_from_result {
     for my $lane (@per_lane_alignments) {
         $total_bases += $lane->total_base_count;
         $total_mapped_bases += $lane->total_aligned_base_count;
-        $mismatches += $lane->total_base_count * $lane->instrument_data->filt_error_rate_avg / 100;
+        my $filt_error_rate_avg;
+        if ($mismatches ne "NA" and $lane->instrument_data->can("filt_error_rate_avg")) {
```

Comment:
Yes, because if any of the lanes doesn't have an error rate, then we don't want to report it at all.